src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -566,49 +575,138 @@ LRESULT CALLBACK FancyZones::s_WndProc(HWND window, UINT message, WPARAM wparam,
 
 void FancyZones::UpdateZoneWindows() noexcept
 {
-    auto callback = [](HMONITOR monitor, HDC, RECT *, LPARAM data) -> BOOL

```

Comment:
Add a comment:
```
// Callback for EnumDisplayMonitors()
```

Reply:
@enricogior I'll rename `callback` to `EnumDisplayMonitorsCallback` if you don't mind